src/crypto/crypto.cc

```diff
@@ -120,57 +111,48 @@ Base64Key::Base64Key( string printable_key )
     throw CryptoException( "Key must be well-formed base64." );
   }
 
-  if ( len != 16 ) {
-    throw CryptoException( "Key must represent 16 octets." );
-  }
+  if ( len != 16 ) { throw CryptoException( "Key must represent 16 octets." ); }
 
   /* to catch changes after the first 128 bits */
   if ( printable_key != this->printable_key() ) {
     throw CryptoException( "Base64 key was not encoded 128-bit key." );
   }
 }
 
-Base64Key::Base64Key()
-{
-  PRNG().fill( key, sizeof( key ) );
-}
+Base64Key::Base64Key() { PRNG().fill( key, sizeof( key ) ); }
 
-Base64Key::Base64Key(PRNG &prng)
-{
-  prng.fill( key, sizeof( key ) );
-}
+Base64Key::Base64Key( PRNG &prng ) { prng.fill( key, sizeof( key ) ); }
 
 string Base64Key::printable_key( void ) const
 {
   char base64[ 24 ];
-  
+
   base64_encode( key, 16, base64, 24 );
 
-  if ( (base64[ 23 ] != '=')
-       || (base64[ 22 ] != '=') ) {
-    throw CryptoException( string( "Unexpected output from base64_encode: " ) + string( base64, 24 ) );
+  if ( ( base64[ 23 ] != '=' ) || ( base64[ 22 ] != '=' ) ) {
+    throw CryptoException( string( "Unexpected output from base64_encode: " )
+                           + string( base64, 24 ) );
   }
 
   base64[ 22 ] = 0;
   return string( base64 );
 }
 
 Session::Session( Base64Key s_key )
-  : key( s_key ), ctx_buf( ae_ctx_sizeof() ),
-    ctx( (ae_ctx *)ctx_buf.data() ), blocks_encrypted( 0 ),
-    plaintext_buffer( RECEIVE_MTU ),
-    ciphertext_buffer( RECEIVE_MTU ),
-    nonce_buffer( Nonce::NONCE_LEN )
+  : key( s_key )
+  , ctx_buf( ae_ctx_sizeof() )
+  , ctx( (ae_ctx *)ctx_buf.data() )
+  , blocks_encrypted( 0 )
+  , plaintext_buffer( RECEIVE_MTU )
+  , ciphertext_buffer( RECEIVE_MTU )
+  , nonce_buffer( Nonce::NONCE_LEN )
```

Comment:
I don't think we want leading-comma for this

Reply:
Fixed.